src/platform/driver_init.c

```diff
@@ -198,6 +198,7 @@ static void _mci_init(void)
         GCLK, SDHC0_GCLK_ID, CONF_GCLK_SDHC0_SRC | (1 << GCLK_PCHCTRL_CHEN_Pos));
     hri_gclk_write_PCHCTRL_reg(
         GCLK, SDHC0_GCLK_ID_SLOW, CONF_GCLK_SDHC0_SLOW_SRC | (1 << GCLK_PCHCTRL_CHEN_Pos));
+    mci_sync_init(&MCI_0, SDHC0);
```

Comment:
Why is this newly needed?

Reply:
We put that line in sd_mmc.c:930, but this is a line that belongs to user code and not driver code. I noticed that I missed removing it from sd_mmc.c when I reverted that file. And then my autoformatter also made it harder to see the diff. Should be better now.